src/components/Modal/index.tsx

```diff
@@ -125,6 +125,9 @@ class Modal extends React.Component<ThemedModalProps, any> {
 				responsive={false}
 				margin="small"
 				animate={false}
+				style={{
+					marginTop: theme.header.height,
```

Comment:
Let's apply it only when the position is `top`, otherwise it's fine to use style, I don't mind it.